invenio_oaiserver/response.py

```diff
@@ -257,7 +254,10 @@ def listidentifiers(**kwargs):
 
 def listrecords(**kwargs):
     """Create OAI-PMH response for verb ListIdentifiers."""
-    from invenio_records.models import RecordMetadata
+    kwargs['verb'] = 'ListRecords'
```

Comment:
we always pass validated arguments from `views/server.py` and they **always** contain `verb`.